xml/OptionPanel.xml

```diff
@@ -1,108 +1,182 @@
 <!-- Default formatting -->
 <Defaults>
-  <Text color="white" alignment="MiddleLeft"/>
+  <Text color="#FFFFFF" alignment="MiddleLeft"/>
   <Toggle isOn="False" rectAlignment="MiddleRight"/>
+  <Cell dontUseTableCellBackground="true" outlineSize="0 1" outline="grey"/>
 
-  <VerticalLayout class="window" visibility="Admin" active="false" color="black" width="500" height="700" outlineSize="1 1" outline="grey" showAnimation="SlideIn_Right" hideAnimation="SlideOut_Right" animationDuration="0.1"/>
-  <VerticalLayout class="group" visibility="Admin" outlineSize="1 1" outline="grey" color="#222222" padding="10" spacing="5"/>
-  <VerticalLayout class="text-column" visibility="Admin" padding="5 20 0 0"/>
+  <!-- main window -->
+  <TableLayout class="window" width="500" height="700" active="false" color="#000000" outlineSize="2 2" outline="grey" showAnimation="SlideIn_Right" hideAnimation="SlideOut_Right" animationDuration="0.1"/>
```

Comment:
Don't have to do it in this PR, but please consider a format pass here.  It's common with XML to split attributes to different lines, which greatly enhances readability.  One attribute is probably fine to do inline, but 2 or more should be on separate lines.

Reply:
Sure, can do so for the next update.